Makefile

```diff
@@ -94,7 +94,7 @@ local-ecr-login:
 .PHONY: local-hostconfig
 local-hostconfig:
 	if [ "$$(uname -s)" == "Darwin" ]; then \
-	  sudo ./scripts/happy hosts install; \
+	  sudo -E ./scripts/happy hosts install; \
```

Comment:
related-ish - preserves env variables, ensures venv python works for me (although this was not necessary for others -- would be nice if it were confirmed that this recipe still works for you)

Reply:
`make local-hostconfig` and `make local-nohostconfig` appear to work for me. I'm using `pyenv virtualenv`.

Reply:
yep, understood. This helps it work for more people such as myself; my setup requires it for some reason. But conversed with @ebezzi , going to set these changes aside and have others review.